db/migrate/20170209211017_join_authentication_orchestration_stack.rb

```diff
@@ -0,0 +1,9 @@
+class JoinAuthenticationOrchestrationStack < ActiveRecord::Migration[5.0]
+  def change
+    create_table :authentication_orchestration_stacks do |t|
+      t.bigint :authentication_id
+      t.bigint :orchestration_stack_id
+      t.index  :orchestration_stack_id, :name => "index_authentication_orchestration_stacks_o.stack_id"
```

Comment:
should be probably:

t.index  [:authentication_id, :orchestration_stack_id], :unique => true, :name => "index_authentication_orchestration_stacks"

?